modules/upgrade/pages/k-rolling-upgrade.adoc

```diff
@@ -95,111 +80,49 @@ curl -s 's://hub.docker.com/v2/repositories/redpandadata/redpanda/tags/?ordering
-
-. Check for topics that have a replication factor greater than one.
-+
-If you have topics with `replication.factor=1`, and if you have sufficient disk space, Redpanda Data recommends temporarily increasing the replication factor. This can help limit outages for these topics during the rolling upgrade. Do this before the upgrade to make sure there's time for the data to replicate to other brokers. For more information, see xref:manage:data-migration.adoc#change-topic-replication-factor[Change topic replication factor].
-
-. <<Deploy an upgraded StatefulSet>> with your desired Redpanda version.
-. <<Upgrade and restart the brokers>> separately, one after the other.
-
-[WARNING]
-====
-Redpanda Data does not recommend using the `kubectl rollout restart` command to perform rolling upgrades.
-Although the chart's `preStop` lifecycle hook puts the broker into maintenance mode before a Pod is deleted,
-the `terminationGracePeriod` may not be long enough to allow maintenance mode to finish.
-If maintenance mode does not finish before the Pod is deleted, you may lose data.
-After the `terminationGracePeriod`, the container is forcefully stopped using a `SIGKILL` command.
-
-If you want to use `kubectl rollout restart`, it can be a challenge to determine the necessary value for the `terminationGracePeriod`. In common cases, 30 seconds should be sufficient.
-For large clusters, 90 seconds should be sufficient. You can test different values in a development environment. To configure the `terminationGracePeriod`,
-use the xref:reference:redpanda-helm-spec.adoc#statefulsetterminationgraceperiodseconds[`statefulset.terminationGracePeriodSeconds`] setting.
-====
-
-=== Deploy an upgraded StatefulSet
+A rolling upgrade involves running the following procedure on each broker one at a time:
 
-To deploy an upgraded StatefulSet, you need to delete the existing StatefulSet, then upgrade the Redpanda Helm chart deployment with your desired Redpanda version.
+. Upgrade the version of Redpanda on the broker that's running in the Pod with the highest ordinal.
+. Place the broker into maintenance mode.
+. Wait for maintenance mode to finish.
+. Terminate the Pod so that it is recreated with the new version of Redpanda.
+. Take the broker out of maintenance mode.
 
-. Delete the existing StatefulSet, but leave the Pods running:
-+
-```bash
-kubectl delete statefulset redpanda --cascade=orphan --namespace <namespace>
-```
+Placing brokers into maintenance mode ensures a smooth upgrade of your cluster while reducing the risk of interruption or degradation in service. When a broker is placed into maintenance mode, it reassigns its partition leadership to other brokers for all topics that have a replication factor greater than one. Reassigning partition leadership involves _draining_ leadership from the broker and _transferring_ that leadership to another broker.
```

Comment:
Isn't the minimum recommended replication factor 3? I've seen posts in Community where this has tripped up users. Wondering if you should mention?
"...a replication factor greater than one (3 is the default)."